tigerpath/majors_and_certificates/majors/CLA_2018.json

```
@@ -0,0 +1,373 @@
+                    "POL 301",
+                    "ART 205"
+                  ]
+                }
+              ]
+            }
+          ]
+        },
+        {
+          "name": "Classical Studies",
+          "max_counted": 1,
+          "min_needed": "ALL",
+          "explanation": "This track offers the opportunity for sustained and focused inquiry into the history, literature, and culture of the ancient Mediterranean, as well as the impact of classical antiquity on later periods by using a variety of interpretative methods. The particular program for each student is determined in collaboration with the Director of Undergraduate Studies (DUS) and/or a faculty adviser. The focus can be on a specific disciplinary subfield (e.g., ancient politics) or on a particular period to be explored from a number of perspectives (e.g., the history, literature, and art of Imperial Rome). Each track must have a methodological component designed to introduce the student to techniques of analysis appropriate to the student's particular interests. This component of the track is satisfied by two comparative or methodological courses, the subject matter of which is concerned primarily with the classical world. These courses are chosen by each student in consultation with a faculty adviser and/or the DUS and must be preapproved by the DUS.",
+          "req_list": [
+            {
+              "name": "Prerequisites",
+              "max_counted": 1,
+              "min_needed": 1,
+              "explanation": "At least one course in the Department of Classics or cross-listed as CLA. A second course in addition to the prerequisite is strongly recommended, as well, but this course can count as one of the six departmentals focusing on classical civilization. A freshman seminar on a classical subject may count as a prerequisite, but may not be used as a departmental.\n\nApplicants to this track must submit a proposed plan of study, worked out in consultation with the DUS, by the end of Fall term in their junior year. Given the range of possible interests (e.g., Greek and Roman culture, Late Antique or Byzantine/Mediaeval literature, the reception of antiquity in later periods and cultures), each applicant may bring to the study of the ancient world, there is no set list of fields of concentration, and faculty members can give additional guidance in preparing a track of study.",
+              "course_list": [
+                "CLA *"
+              ]
+            },
+            {
+              "name": "Ancient Greek or Latin",
+              "max_counted": 1,
+              "min_needed": 1,
+              "explanation": "Each student must successfully complete either Ancient Greek or Latin to the level of 108 or achieve an equivalent level of knowledge, as demonstrated through test scores (SAT, AP), a placement exam, or coursework (including through a summer language program). However, at least one language course must be taken at Princeton.",

```

Why not also include their warning.
```suggestion
              "explanation": "Each student must successfully complete either Ancient Greek or Latin to the level of 108 or achieve an equivalent level of knowledge, as demonstrated through test scores (SAT, AP), a placement exam, or coursework (including through a summer language program). However, at least one language course must be taken at Princeton.\n\nStudents should be aware that most graduate programs in Classics will demand a more extensive training in the ancient languages than these minimum requirements. Those considering going on to graduate school should plan to do additional work in Greek and Latin either during their time at Princeton or through a postbaccalaureate program.",
```